scripts/dap-ami-bootstrap.sh

```diff
@@ -0,0 +1,56 @@
+#!/bin/bash
+set -e
+
+amazon-linux-extras install -y docker
+systemctl enable docker
+systemctl start docker
+
+if ! docker images | grep appliance; then
+  echo "Loading DAP appliance image into Docker"
+  docker load -i /opt/dap-appliance.tar.gz
+fi
+
+image_id=$(docker images -q)
+container_name='dap'
+
+docker rm -f ${container_name} > /dev/null 2>&1 || true
```

Comment:
WDYT on deleting also containers that created by the image or having ports 443 open (will conflict with invoking new DAP container)

Reply:
This is deleting the container, not the image.

Reply:
> This is deleting the container, not the image.
I know, didn't mention deleting images..

Reply:
Then I'm confused. Could you elaborate on your suggestion?